templates/create-daml-app/package.json.template

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
With the version of `daml2ts` on `master`, we don't need yarn workspaces at all anymore. Instead, the `ui/package.json` should reference `file:../daml-ts/create-daml-app-x.y.z`